File: Lab1/app/src/main/java/com/example/lab1/StudentInfo.java
package com.example.lab1;

import java.io.Serializable;

public class StudentInfo implements Serializable {
    public String fullName, studentID, className, schoolYear, yearLevel, departments, plan, phoneNumber;

    public StudentInfo(String fullName, String studentID, String className, String schoolYear,
                       String yearLevel, String departments, String plan, String phoneNumber) {
        this.fullName = fullName;
        this.studentID = studentID;
        this.className = className;
        this.schoolYear = schoolYear;
        this.yearLevel = yearLevel;
        this.departments = departments;
        this.plan = plan;
        this.phoneNumber = phoneNumber;
    }
}

File: Final_Project/app/src/main/java/com/example/testapi2/model/StudentInfo.kt
package com.example.testapi2.model


import java.io.Serializable

data class StudentInfo(
    val fullName: String,
    val studentID: String,
    val className: String,
    val schoolYear: String,
    val yearLevel: String,
    val departments: String,
    val plan: String,
    val phoneNumber: String
) : Serializable
